lib/logstash/filters/jdbc_static.rb

```diff
@@ -176,7 +180,9 @@ def loader_runner
   def prepare_data_dir
     # later, when local persistent databases are allowed set this property to LS_HOME/data/jdbc-static/
     # must take multi-pipelines into account and more than one config using the same jdbc-static settings
-    java.lang.System.setProperty("derby.system.home", ENV["HOME"])
+    path_data = Pathname.new(@settings.get_value("path.data")).join("plugins", "filter", "jdbc_static")
```

Comment:
`@settings` can be `nil` (per other comments, on LS5?), which will cause this line to throw a `NoMethodError`.

Reply:
Fixed by c3fc4008165d65438928fc32f72208d68eaf6ae3